BREAKING.md

```diff
@@ -15,11 +15,11 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 - Avoid using code formatting in the title (it's fine to use in the body).
 - To explain the benefit of your change, use the [What's New](https://fluidframework.com/docs/updates/v1.0.0/) section on FluidFramework.com.
 
-# 2.0.0
+# 2.0.0-internal.2.0.0
 
-## 3.0.0 Upcoming changes
-- [Signature from  `ISummarizerInternalsProvider.refreshLatestSummaryAck` interface has changed](#Change-ISummarizerInternalsProvider.refreshLatestSummaryAck-interface)
-- [Move `TelemetryNullLogger` and `BaseTelemetryNullLogger` to telemetry-utils package](#Move-`TelemetryNullLogger`-and-`BaseTelemetryNullLogger`-to-telemetry-utils-package)
+## 2.0.0-internal.2.0.0 Upcoming changes
+- [Signature from  ISummarizerInternalsProvider.refreshLatestSummaryAck interface has changed](#Change-ISummarizerInternalsProvider.refreshLatestSummaryAck-interface)
```

Comment:
Is this a breaking change?

Reply:
Looks like there is a mismatch in the name. Below the note is called `### Change-ISummarizerInternalsProvider.refreshLatestSummaryAck-interface`. It does look like it is a breaking change though